templates/workspaces/base/porter.yaml

```diff
@@ -86,6 +86,10 @@ parameters:
       "The client secret of the workspace in the identity provider. This value is typically provided to you
       when you create the ws application"
     default: ""
+  - name: ui_client_id
+    type: string
+    default: ""
+    description: "The client ID of the UI"
   - name: scope_id
     type: string
```

Comment:
Hi, the unrestricted and the airlock import review workspace templates are based on the base workspace template.  I think their creation is now failing because they don't have ui_client_id and auto_grant_workspace_consent specified in their porter.yaml. I also think we should also add a note in the change log  noting for customers who might use the same pattern (inherit the base and extend it) to add those parameters

Reply:
Perfect thanks for the review. I'll get on that.